### PR TITLE
Add ViewString for IsHomalgRingElement

### PR DESCRIPTION
**Pull request by zickgraf**
For example, this is used by RingAsCategory.

### CODE REVIEW

File: MatricesForHomalg/gap/HomalgRing.gi [hunk 1]
```diff
@@ -2891,6 +2891,17 @@ InstallMethod( ViewObj,
     
 end );
 
+##
+InstallMethod( ViewString,
+        "for homalg ring elements",
+        [ IsHomalgRingElement ],
+        
+  function( o )
```
Comment (mohamed-barakat) [inline MatricesForHomalg/gap/HomalgRing.gi:2899]: I would prefer such a code

```
InstallMethod( ViewString,
        "for homalg ring elements",
        [ IsHomalgRingElement ],

  Name );
```

**Review by mohamed-barakat — COMMENTED** [review]
Would it make sense to bump the version?

File: MatricesForHomalg/gap/HomalgRing.gi (hunk 1 above)
Comment (zickgraf) [reply MatricesForHomalg/gap/HomalgRing.gi]: Makes sense, I have pushed an updated version. Although I really hate it if I quickly want to put a "Display" or an "Error" somewhere for debugging and come across a method installation like this :D

**Comment by zickgraf** [comment]
> Would it make sense to bump the version?

Instead of thinking about that question I have bumped the version :-)

**Pull request merged**